spec/features/collections/collection_permissions_spec.rb

```diff
@@ -1,4 +1,5 @@
-describe 'Collections permissions', js: true do
+# EDL Failed Test
```

Comment:
I am sure if you either want to include # EDL Failed Test or you forgot to remove.